### DISCUSSION TITLE
Pydantic not enforcing maximum column length

### DISCUSSION THREAD

**Discussion by ajawu** [tiangolo/pydantic-sqlalchemy#80] [Questions] [answered]
`
class User(db.Model):
    __tablename__ = 'user'
    id = db.Column(db.Integer, autoincrement=True, primary_key=True)
    first_name = db.Column(db.String(40), nullable=False)
    last_name = db.Column(db.String(40), nullable=False)

UserSchema = sqlalchemy_to_pydantic(User, exclude=['id'])

UserSchema doesn't enforce the maximum length on the first_name and last_name fields.
`

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile:

This would be a feature request.

I deprecated this library a while ago in favor of SQLModel which would solve all the same use cases in a much better way, and I'm directing all the effort in that direction.

SQLModel uses Pydantic models to define both data and table (SQL) models, including that it enforces max length. :rocket:
